docs/src/pages/_document.page.tsx

```diff
@@ -41,7 +41,7 @@ const getCSPContent = (context: Readonly<HtmlProps>) => {
       ...ANALYTICS_CSP.all.frame,
       ...ANALYTICS_CSP.prod.frame,
     ].join(' ')} *.youtube.com;
-    img-src 'self' blob: ${[
+    img-src 'self' blob: data: ${[
```

Comment:
Can we lock this down to that specific data image?

Reply:
Great question! Unfortunately, no. It is a schema and you can't put anything behind the `:`. In general it sounds like data schema is bad to allowlist, BUT it sounds like maybe it isn't so bad for `img-src` : https://security.stackexchange.com/questions/94993/is-including-the-data-scheme-in-your-content-security-policy-safe/95011#95011

We could root cause this and fix whatever Geo code has a side effect that is running, but that sounds like a lot of work.